index.js

```diff
@@ -72,19 +75,21 @@ const axios = require("axios");
 
       out.specs.push({
         fileName,
-        oas,
+        oas: JSON.parse(oas),
```

Comment:
I think because `api.config.json` is completely undocumented functionality right now, this doesnt matter. For the common use case of us finding a spec file, packaging it up and sending it, this should suffice. 

I still need to do an e2e test with this branch though on micro, just to make sure it's still working with this change. I made the change to get the tests to pass without the spec file being double encoded with a bunch of spaces in it.